ambari-infra-manager/src/main/resources/log4j2.xml

```diff
@@ -24,7 +24,7 @@
       <Layout type="PatternLayout" pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36}:%L - %msg%n" />
     </Appender>
     <RollingFile name="File" fileName="${logging.file}" filePattern="${logging.file}-%i-%d{yyyy-MM-dd}">
-      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36}:%L - %msg%n" />
+      <PatternLayout pattern="%d [%t] %-5p %C{6} (%F:%L) - %m%n" />
```

Comment:
is %d good enough as default? %d could be depdends on system settings

Reply:
Changed to `%d{ISO8601}`